From 76e526ca9b5f395745778ecc0d2dc4ce7461c7ac Mon Sep 17 00:00:00 2001
From: Jan Synacek <jsynacek@redhat.com>
Date: Wed, 27 Mar 2019 12:58:51 +0100
Subject: [PATCH 1/2] build-sys: make rdisc.service a regular unit

There is no need to output it as a template unit, as there is no
template specific syntax in the unit file.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 97c4a12..8bb5756 100644
--- a/meson.build
+++ b/meson.build
@@ -288,7 +288,7 @@ if build_rinfod == true
 		subs.set('sbindir', join_paths(get_option('prefix'), get_option('sbindir')))
 		unit_file = configure_file(
 			input: 'systemd/rdisc.service.in',
-			output: 'rdisc@.service',
+			output: 'rdisc.service',
 			configuration: subs
 		)
 		install_data(unit_file, install_dir: systemdunitdir)
-- 
2.20.1

